styles.css

```diff
@@ -3,57 +3,72 @@
 body {
   background-color: gainsboro;
```

Comment:
-Kindly remove the `background-color: gainsboro;` it adds the extension color to the top of your navigation which is not needed. kindly fix that it will make your design match more to the Figma design as required in the project :sunglasses: :sunglasses: :sunglasses: 
 check below your design compared to the Figma design
 
##  your design  :sunglasses: 

![image](https://user-images.githubusercontent.com/95297251/193125654-c714c3dc-9e0a-4fe2-a04e-b9ab079bf5f9.png)

## Figma design :+1: 

![image](https://user-images.githubusercontent.com/95297251/193126071-81478059-4fe6-40ff-8c98-c3e2de96244d.png)